app/components/PartialChallengeDetailsView.js

```diff
@@ -91,15 +66,51 @@ PartialChallengeDetailsView.propTypes = {
 }
 
 const styles = StyleSheet.create({
```

Comment:
I think both ways have pros and cons so I'm fine with either way. 

My personal thoughts: I think as long as we are decomposing our components properly, the Stylesheets shouldn't be too large and therefore having the Stylesheet inside of the file is fine. Plus, it's nice to be able to move around a single file if necessary without having dependencies on other files. 

However, shared css make an excellent candidate for external stylesheets. As such, we may consider adding a "css" folder inside of the "assets" folder so that we can have common styles for buttons, fonts, etc. that are shared across components. Once we go back to add styles, I would anticipate that the much of our css will be common styles shared across components and this will be necessary.